x-pack/plugins/apm/public/components/app/service_inventory/service_list/index.tsx

```diff
@@ -136,16 +130,10 @@ export function getServiceColumns({
       }),
       sortable: true,
       render: (_, { serviceName, agentName, transactionType }) => (
-        <TruncateWithTooltip
-          data-test-subj="apmServiceListAppLink"
-          text={formatString(serviceName)}
-          content={
-            <ServiceLink
-              agentName={agentName}
-              query={{ ...query, transactionType }}
-              serviceName={serviceName}
-            />
-          }
+        <ServiceLink
```

Comment:
Don't we want to show a tooltip with the entire service name in case it is truncated?

Reply:
Ah, I see that you moved it to the `ServiceLink`.